README.md

```diff
@@ -256,12 +256,175 @@ To access the data from the PaymentAck just do:
 And that will return a dictionary with all of the fields of the PaymentACK and the values that were 
 filled in.
 
-[1]: https://github.com/bitcoin/bips/blob/master/bip-0070.mediawiki
-[2]: https://github.com/bitcoin/bips/blob/master/bip-0075.mediawiki
-[3]: https://mvnrepository.com/artifact/com.netki/transactid
+## Key Management system
 
----
 ## Vault for Key Storage
 This library includes integration with Hashicorp Vault for key storage using the key/value secrets engine and can be launched as a Docker container.
 See Hashicorp's Vault Docker documentation here: https://hub.docker.com/_/vault
 
+## General Usage
+
+To use the methods to use the key management you can use the static methods in the class TidKms for example:
```

Comment:
To use the key management integration you can use the static methods in the class TidKms, for example: